File: cpp_e9zMZ/cpp/04-dayso/99.cpp
#include <math.h>

#include <iostream>
using namespace std;

int main() {
    int x = 6;
    int sum = 0;
    string result = "";

    for (int i = 1; i <= x; i++) {
        int power = 2 * i;
        sum += pow(x, power);
        string sign = i == x ? " = " : " + ";
        result += to_string(x) + "^" + to_string(power) + sign;
    }

    cout << result << sum;
    return 0;
}

File: python_ToSXs/python/04-dayso/99.py
n = 4

result = ""
zsum = 0
for i in range(1, n + 1):
    zsum += pow(n, i * 2)
    result += f"{n}^{i * 2}" + " + "

result = result[0:len(result) - 3]
print(f"{result} = {zsum}")
